src/nl/hannahsten/texifyidea/lang/commands/LatexDiffcoeffCommand.kt

```diff
@@ -0,0 +1,38 @@
+package nl.hannahsten.texifyidea.lang.commands
+
+import nl.hannahsten.texifyidea.lang.LatexPackage
+import nl.hannahsten.texifyidea.lang.LatexPackage.Companion.DIFFCOEFF
+
+/**
+ * @author Florian Kraft
+ */
+enum class LatexDiffcoeffCommand(
+    override val command: String,
+    override vararg val arguments: Argument = emptyArray(),
+    override val dependency: LatexPackage = LatexPackage.DEFAULT,
+    override val display: String? = null,
+    override val isMathMode: Boolean = true,
+    val collapse: Boolean = false
```

Comment:
This can also be removed

Reply:
Do you mean remove the lines display, isMathMode and collapse completely? I am wondering, since every command file has these lines

Reply:
No sorry just the `collapse`, as it is unused